docs/source/markdown/podman-mount.1.md.in

```diff
@@ -41,31 +41,32 @@ Do not truncate the output (default *false*).
 
 ## EXAMPLE
 
+In rootful mode, Mount specified container.
 ```
-podman mount c831414b10a3
-
+# podman mount c831414b10a3
 /var/lib/containers/storage/overlay/f3ac502d97b5681989dff84dfedc8354239bcecbdc2692f9a639f4e080a02364/merged
 ```
 
+In rootless mode, containwer mounting only works after executing the podman unshare command to enter the user namespace.
```

Comment:
typo, "containwer"

Reply:
Feels a little too verbosely verbose. Is the "podman unshare" part really necessary? What would you think of
> ...mounting only works from inside the user namespace.